stixcore/processing/levelb.py

```diff
@@ -0,0 +1,150 @@
+import os
+import logging
+from time import perf_counter
+from pathlib import Path
+
+import numpy as np
+
+from astropy.table import unique, vstack
+from astropy.table.table import QTable, Table
+
+from stixcore.io.fits.processors import SEC_IN_DAY, FitsLBProcessor
```

Comment:
Can probably replace this with astroy units `(1 * u.d).to_value('s')` but not sure if we should

Reply:
not done at the moment